schemas/org.youtube/youtube/jsonschema/1-0-0

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a youtube player event",
+  "self": {
+    "vendor": "org.youtube",
+    "name": "youtube",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "avaliablePlaybackRates": {
+      "type": "array",
+      "description": "An array of playback rates in which the current video is available",
+      "items": {
+        "type": "number",
+        "minimum": 0,
+        "maximum": 9007199254740991
+      }
+    },
+    "cued": {
+      "type": "boolean",
+      "description": "If the video is cued"
+    },
+    "playerId": {
+      "type": "string",
+      "description": "The HTML id of the video element",
+      "maxLength": 65535
+    },
+    "autoPlay": {
+      "type": "boolean",
+      "description": "This specifies whether the initial video will automatically start to play when the player loads."
+    },
+    "buffering": {
+      "type": "boolean",
+      "description": "If the player is buffering"
+    },
+    "controls": {
+      "type": "boolean",
+      "description": "Whether the video player controls are displayed"
+    },
+    "error": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "An string of the latest error to occur, or null if no errors",
+      "enum": [
+        "INVALID_PARAMETER",
+        "HTML5_PLAYER_ERROR",
+        "NOT_FOUND",
+        "EMBED_DISALLOWED"
+      ]
+    },
+    "loaded": {
+      "type": "number",
+      "description": "The percentage of the video that the player shows as buffered",
+      "minimum": 0,
+      "maximum": 1
+    },
+    "origin": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The origin domain of the embed",
+      "maxLength": 65535
+    },
+    "playlist": {
+      "type": [
+        "array",
+        "null"
+      ],
+      "description": "An array of the video IDs in the playlist as they are currently ordered."
+    },
+    "playlistIndex": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "The index of the playlist video that is currently playing",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "unstarted": {
+      "type": "boolean",
+      "description": "If the player hasn't started"
+    },
+    "url": {
+      "type": "string",
+      "description": "The YouTube embed URL of the media resource",
```

Comment:
format uri?

Reply:
It will always be a URL (as it will be a YouTube.com link).
https://developers.google.com/youtube/iframe_api_reference#Retrieving_video_information